LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadDataLns.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools.LoRaMessage
+{
+    using System;
+    using System.Buffers.Binary;
+    using LoRaTools.Utils;
+    using LoRaWan;
+
+    public class LoRaPayloadDataLns : LoRaPayloadData
+    {
+        public LoRaPayloadDataLns(DevAddr devAddress,
+                                  MacHeader macHeader,
+                                  FrameControl control,
+                                  ushort counter,
+                                  string options,
+                                  string payload,
+                                  FramePort port,
+                                  Mic mic)
+        {
+            if (string.IsNullOrEmpty(payload)) throw new ArgumentNullException(nameof(payload));
+
+            // Writing the DevAddr
+            DevAddr = new byte[LoRaWan.DevAddr.Size];
+            _ = devAddress.Write(DevAddr.Span);
+            DevAddr.Span.Reverse();
```

Comment:
<iframe src="https://giphy.com/embed/NEvPzZ8bd1V4Y" width="459" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/editingandlayout-NEvPzZ8bd1V4Y">via GIPHY</a></p>

Reply:
It doesn't work, but still I am proud